README.md

```diff
@@ -1,24 +1,59 @@
-ros-install-osx   [![Build Status](https://travis-ci.org/mikepurvis/ros-install-osx.svg?branch=master)](https://travis-ci.org/mikepurvis/ros-install-osx)
+ros-install-osx   
 ===============
 
-This repo aims to maintain a usable, scripted, up-to-date installation procedure for
-[ROS](http://ros.org), currently Lunar. The intent is that the `install` script may
-be executed on a El Capitan or newer machine and produce a working desktop_full
-installation, including RQT, rviz, and Gazebo.
+* This commonly breaks depending on what gets pushed over brew and whether the corresponding packages update deprecated code. Efforts have been made to pick out working versions of specific packages, but nothing is guaranteed.
 
-This is the successor to my [popular gist on the same topic][1].
+This repo aims to maintain a usable, scripted, up-to-date installation procedure for [ROS](http://ros.org), currently melodic. The intent is that the `install` script may be executed on a Catalina or newer machine and produce a working desktop_full  installation, including RQT, rviz, and Gazebo.
 
-[1]: https://gist.github.com/mikepurvis/9837958
+This is the successor to the [popular gist on the same topic][1]. Also thanks to [Boris Gromov](https://gist.github.com/bgromov) for [his helpful gist](https://gist.github.com/bgromov/23a74bbe846d965964b150080cb2d574).
 
+## Current Status
+
+**Note: This should work as of Oct 6, 2019 on Catalina GM** 
+
+Required software versions (installed via script):
+
+* **boost**: 1.69
+* **opencv3**
+* **python2**
+* **sip**: 4.19.8_12 - Required for python2 support
+* **pyqt**: 5.10.1_1 - Required for python 2 support
```

Comment:
How realistic would it be to have Python 3 as the default now? We're using it for most of the base system at CPR, so I'd expect it to mostly work.

Reply:
I'm not sure. Isn't ROS planning on officially adopting it as of ROS "noetic" which isn't ready yet? I'll play around with it in a couple months before python 2 support officially gets dropped.